src/App.js

```diff
@@ -1,13 +1,15 @@
+/* eslint-disable */
 import React from 'react';
 import './App.css';
-import NewCalculator from './components/newCalculator';
-
+import Home from './Home';
+import Headers from './components/Headers';
 // eslint-disable-next-line react/prefer-stateless-function
 class App extends React.Component {
   render() {
     return (
       <div className="App">
-        <NewCalculator />
+         <Headers />
+       <Home/>
       </div>
     );
   }
```

Comment:
- [ ] Good job so far but I believe you can do better with the design. You can search the internet for ideas. Try to add more colours and animations. [optional]

Reply:
alright. I would do better with the design. Thank you for all other suggestions too. Would put it into considerations

Reply:
👍🏽